reference/configuration/doctrine.rst

```diff
@@ -499,4 +499,135 @@ can be placed directly under ``doctrine.orm`` config level.
 This shortened version is commonly used in other documentation sections.
 Keep in mind that you can't use both syntaxes at the same time.
 
+Custom Mapping
+--------------
+
+It's possible to not rely on the auto mapping and set all the configuration
+on your own.
+
+For example, you can overwrite the configuration directory for a given bundle.
+Keep in mind that the bundle you want to overwrite has to exist.
+
+.. configuration-block::
+
+    .. code-block:: yaml
+    
+        doctrine:
+            # ...
+            orm:
+                # ...
+                auto_mapping: true
+                mappings:
+                    # ...
+                    AppBundle:
+                        dir: Resources/config/doctrine
```

Comment:
I know *now* that `AppBundle` *is* significant here - because it is a real bundle, and because is_bundle is true (the default), the `dir` is relative to the bundle. Gosh, that's kind of confusing :). I would break this section into 2 parts:

A) Mapping Entities in a Bundle
B) Mapping Entities outside of a Bundle

That might help make it clear